prototyping/xarray/xdose.py

```diff
@@ -221,3 +226,112 @@ def _validate_coord_system(coord_system):
                 "\n".join(VALID_COORD_SYSTEMS)
             )
         )
+
+
+def plot_xdose_tcs_at_point(xdose_to_plot, point):
```

Comment:
Given this is within the `xdose` module, probably could get away with not having xdose in the name. For example, when xdose is imported it would look like this:

```python
import xdose

xdose.plot_tcs_at_point( ... )
```